apps/health_campaign_field_worker_app/lib/pages/beneficiary_registration/school/school_individual_details.dart

```diff
@@ -808,9 +812,26 @@ static const _individualLastNameKey = 'individualLastName';
                   _heightKey,
                   height.length == 1 ? '0$height' : height,
                 ),
+                const AdditionalField(
+                  _type,
+                  "SCHOOL",
+                ),
+                 AdditionalField(
+                  _schoolName,
```

Comment:
key name for this is schoolName but mapping parent value,

it should 2 different additional fields